build/rbac/rbac.yaml

```diff
@@ -0,0 +1,1239 @@
+kind: ClusterRole
```

Comment:
Other generated resources like crds.yaml are just generated directly to the location where they are needed such as the examples folder. Perhaps in a future PR, will this be directly generated directly to common.yaml?

Reply:
That's the idea. For now I want to have some room to modify the helm charts and massage things into place in smaller PRs, then start generating `common.yaml` once things are in good shape rather than trying to change too much at once.